### PR TITLE
Fix build breakage due to ZCP Holds and Bookmarks API in ZoL

### PR DESCRIPTION
**Pull request by sdimitro**
zfs-precommit: http://platform.jenkins.delphix.com/job/devops-gate/job/master/job/zfs-precommit/4717/  (pending)

### CODE REVIEW

**Comment by sdimitro** [comment]
cc @ahrens @pcd1193182

**Comment by sdimitro** [comment]
cc @pzakha

File: tests/zfs-tests/tests/functional/channel_program/synctask_core/tst.list_user_props.ksh [hunk 1]
```diff
@@ -20,6 +20,9 @@
 # DESCRIPTION:
 #       Listing zfs user properties should work correctly.
 #
+#       Note, that this file tests both zfs.list.user_properties
+#       and it's alias zfs.list.properties.
```
Comment (pzakha) [inline tests/zfs-tests/tests/functional/channel_program/synctask_core/tst.list_user_props.ksh]: nit: `it's` -> `its`

Comment (sdimitro) [reply tests/zfs-tests/tests/functional/channel_program/synctask_core/tst.list_user_props.ksh]: Hey Pavel, do you mind if I keep a note of those changes and open a PR at ZoL that will eventually make it here? and just proceed with the code in this PR as-is?

We've been trying with Matt to reduce the diffs between the two codebases which should also reduce the number of merge conflicts->breaks in the build that we take.

Comment (pzakha) [reply tests/zfs-tests/tests/functional/channel_program/synctask_core/tst.list_user_props.ksh]: Ah okay, I didn't realize those were changes from upstream. In this case, I'm fine with it as-is.

**Pull request merged**